lib/astutils.cpp

```diff
@@ -3647,3 +3647,82 @@ bool isUnevaluated(const Token *tok)
 {
     return Token::Match(tok, "alignof|_Alignof|_alignof|__alignof|__alignof__|decltype|offsetof|sizeof|typeid|typeof|__typeof__ (");
 }
+
+static std::set<std::string> getSwitchExprs(const Token *startbrace, bool &hasDefault)
```

Comment:
thinking loud: an alternative could be to provide the `int64_t` value instead of the std::string? Or do you want the exact name sometime?